### PR TITLE
feat: enable tailwind in react components

### PR DESCRIPTION
**Pull request by pregno**
Adopt tailwind for React components

### CODE REVIEW

File: examples/vite/src/application.tsx [hunk 1]
```diff
@@ -9,17 +13,49 @@ import {
...
+
+        <Link
+          href="https://www.google.com"
+          external={true}
+          noUnderline={true}
+          noVisited={true}
+        >
+          Link
+        </Link>
+
```
Comment (hamza14khan) [inline examples/vite/src/application.tsx:42]: non-blocking: Can we remove these extra lines/spacing? If its intentional let me know.

Comment (pregno) [reply examples/vite/src/application.tsx]: done

File: packages/react/ds/src/link/link.tsx [hunk 2]
```diff
@@ -15,7 +14,6 @@ export function Link({
   noUnderline,
   noVisited,
   external,
-  dark = false,
```
Comment (BogdanDanilescu) [inline packages/react/ds/src/link/link.tsx]: if we don't use the `dark` property anymore I think we also need to `link.schema.ts`

**Review by BogdanDanilescu — APPROVED** [review]

**Review by hamza14khan — APPROVED** [review]
Awesome work!

**Pull request merged**